WordPress/Classes/ViewRelated/Comments/Controllers/CommentDetailViewController.swift

```diff
@@ -989,7 +999,7 @@ private extension CommentDetailViewController {
             self?.buttonAddCommentTapped()
         }
         button.isHidden = true
-        containerStackView.addArrangedSubview(button)
+//        containerStackView.addArrangedSubview(button)
```

Comment:
Is this line supposed to be commented out?

Reply:
This button hasn't been used since 25.8 – removing it.